### PR TITLE
Add ECH Discord Channel to EIP Page Index

### PR DESCRIPTION
**Pull request by xinbenlv**
See preview here: https://xinbenlv.github.io/EIPs

### CODE REVIEW

**Comment by xinbenlv** [comment]
What's the best way to preview this change?

**Comment by Pandapip1** [comment]
Set the github pages branch to `patch-21` in your fork.

File: index.html
Comment (Pandapip1) [inline index.html:8]:
```suggestion
  <a href="https://discord.io/EthCatHerders"><img src="https://dcbadge.vercel.app/api/server/Nz6rtfJ8Cu" alt="Discord"></a>
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

File: index.html
Comment (lightclient) [inline index.html:7]: Should remove this tbh.

Comment (Pandapip1) [reply index.html:7]: Should be in a separate PR tho.

Comment (xinbenlv) [reply index.html:7]: Agree with @Pandapip1 that it might be worthy a separate PR

**Comment by xinbenlv** [comment]
> Set the github pages branch to `patch-21` in your fork.

Is there a link I can take a look directly without configuration? Or do I have to compile the github-pages locally?

**Comment by Pandapip1** [comment]
Just set the GH pages branch in your fork settings. No other action is needed.

**Comment by xinbenlv** [comment]
See preview in http://me.zzn.im/EIPs/

**Review by Pandapip1 — APPROVED** [review]

**Comment by xinbenlv** [comment]
Merging is blocked by unrelated errors.

**Comment by Pandapip1** [comment]
Yes. Please close and re-open this PR.

**Review by Pandapip1 — APPROVED** [review]
@SamWilsn please approve (this PR needs 2 approvals)

**Comment by xinbenlv** [comment]
Merged with `origin/master` to update the branch

**Comment by xinbenlv** [comment]
@Pandapip1 @SamWilsn reapprove?

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**
